analytics.go

```diff
@@ -375,6 +369,16 @@ func (c *client) getMarshalled(msgs []message) ([]byte, error) {
 func (c *client) send(msgs []message, retryAttempt int) {
 	const attempts = 10
 
+	ts := c.now()
+	for i := range msgs {
+		err := msgs[i].setSentAt(ts, c.MaxMessageBytes)
+		if err != nil {
+			c.errorf("%s - %v", err, msgs[i].msg)
+			c.notifyFailure([]message{msgs[i]}, err)
+			break
```

Comment:
rather than break here on error, should we not continue and still try to set `sentAt` on the other messages?

Reply:
[3cccb00](https://github.com/cloudquery/analytics-go/pull/1/commits/3cccb00027231f4976b438380ba901c16612b316)